common/locales/en-US/main.ftl

```diff
@@ -120,15 +121,15 @@ settings-profile = Profile Settings
 settings-general = General Settings
     .overlay = Uplink Overlay
     .overlay-description = Enable the on screen Uplink overlay. This will show active call information, as well as allow you to add custom widgets to your screen.
-    .splash-screen = Splash Screen
-    .splash-screen-description = Disabling the splash screen could speed up load times.
     .app-language = App Language
     .change-language = Change Language
     .theme = Theme
     .theme-description = Change the theme of the app.
     .theme-reset = Reset Theme
     .theme-reset-description = Reset the theme to the default.
     .theme-reset-cta = Clear Theme
+    .dyslexia = Open Dyslexic
+    .dyslexia-description = Open Dyslexic may help some users who suffer from dyslexia, it's a custom font you can enable.
```

Comment:
I did not since I don't know the translations, I think we should instead of adding them to every file, we should fall back to the english version of the translation (I think fluent already supports this)